ui/src/components/AppsTable.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { Table, TableHeader, TableBody } from '@patternfly/react-table';
 import { withRouter } from 'react-router-dom';
+import './Table.css';
```

Comment:
Should this be `AppsTable.css` ?

Reply:
I was thinking we could use it for both tables as we probably want them to be consistent which is why I left the name more generic.